### DISCUSSION TITLE
Access Gallery

### DISCUSSION THREAD

**Discussion by brxndonnn** [MustardChef/WSABuilds#187] [Q&A]
is there a way to access the gallery because i want to paste pictures in there that i wanna use but i cant seem to access or find the gallery and when i want to view the userdata.vhdx it gives me an error and i cant mount the disk for some reason PLEASE HELP!

**Comment by MustardChef** [comment]
@brxndonroux 
Open your browser and in the address bar type:
``wsa://com.android.gallery3d``

**Reply by brxndonnn** [reply]
okay i did but how to i add pictures into the gallery thats on my pc

**Reply by brxndonnn** [reply]
dont worry ive found a way thank for the help!

**Reply by hadikamel** [reply]
> dont worry ive found a way thank for the help!

hi
what was the way?
i need help about it

**Reply by MustardChef** [reply]
State your issue!

**Reply by hadikamel** [reply]
> State your issue!

Hi
I just installed wsa in my laptop and i want access folder of my laptop in google photo app at WSA (or file manager or ... )

i can not do it.
what is the point?
